webdriver/tests/element_click/viewport_boundaries.py

```diff
@@ -0,0 +1,124 @@
+import pytest
```

Comment:
I wonder if we should rename `center_point.py` to `click_point.py` and add those new tests there. That way we have the click point tests all together.